git_machete/git_operations.py

```diff
@@ -337,6 +337,14 @@ def get_remotes(self) -> List[str]:
     def get_url_of_remote(self, remote: str) -> str:
         return self._popen_git("config", "--get", f"remote.{remote}.url").strip()  # 'git remote get-url' method has only been added in git v2.5.1
 
+    def get_organization_and_repository_name_of_remote(self, remote: str) -> Tuple[str, str]:
+        org = self.get_config_attr_or_none(f"machete.github.{remote}.organization")
+        repo = self.get_config_attr_or_none(f"machete.github.{remote}.repository")
+        if org and repo:
+            return org.strip(), repo.strip()
+        else:
+            raise MacheteException(f'Organization and repository name is unset for {remote}')
```

Comment:
@PawelLipski not sure if we want exception here or just returning None is enough, wdyt?

Reply:
Hmm, i think that since this is not mandatory exception might be too much so i personally would just return None

Reply:
Yeah, `None` FTW